libr/util/vector.c

```diff
@@ -230,6 +231,11 @@ R_API void r_pvector_free(RPVector *vec) {
 	free (vec);
 }
 
+R_API void r_pvector_set_free(RPVector *vec, RPVectorFree free) {
```

Comment:
uhm i think this should be only used ini the construcctor . changing the destructor of a vector after being initializied can result on undesired behaviours if we have added an item previously

Reply:
Yeah we don't need it anymore anyway. I removed it.